app/views/sales/_invoice.pdf.haml

```diff
@@ -1,5 +1,8 @@
 - customer = sale.customer.decorate
 - organization = sale.organization.decorate
+- comment = ''
+- if sale.comments.first
+  - comment = sale.comments.first.body
```

Comment:
I'm not sure I think this is a good idea.
I think I would rather add a "comment" database field.

However, sales should still have comments so this PR is still valid.